internal/compiler-bridge/src/main/scala/xsbt/API.scala

```diff
@@ -43,14 +61,14 @@ final class API(val global: CallbackGlobal) extends Compat with GlobalHelpers {
       traverser.apply(unit.body)
       val extractUsedNames = new ExtractUsedNames[global.type](global)
       val allUsedNames = extractUsedNames.extract(unit)
-      def showUsedNames(className: String, names: Iterable[String]): String =
-        s"$className:\n\t${names.mkString(", ")}"
-      debuglog("The " + sourceFile + " contains the following used names:\n" +
-        allUsedNames.map((showUsedNames _).tupled).mkString("\n"))
-      allUsedNames foreach {
-        case (className: String, names: Iterable[String]) =>
-          names foreach { (name: String) => callback.usedName(className, name) }
+
+      debuglog("The " + sourceFile + " contains the following used names:\n " + debugOutput(allUsedNames))
```

Comment:
Since we're changing this code, minor suggestion: could we turn this into interpolated strings? As in `s"The $sourceFile ... "`.